migration/v0.25.x-v0.26.x/upgrade-apps.md

```diff
@@ -2,6 +2,15 @@
 
 ## Steps
 
+1. Create a backup for harbor
+
+    ```
+    ./bin/ck8s ops kubectl sc create job -n harbor harbor-backup-v026 --from=cronjobs/harbor-backup-cronjob
+    ./bin/ck8s ops kubectl sc wait --for=condition=complete job -n harbor harbor-backup-v026 --timeout=-1s
+    ```
+
+1. Login into harbor and check that what will be restored
```

Comment:
Did you intend to specify what to check?
```suggestion
1. Login into harbor and check that <what..?> will be restored
```
or
```suggestion
1. Login into harbor and check what will be restored
```

Reply:
I can give some examples :+1:

Reply:
I updated the text to have some examples.